src/App.js

```diff
@@ -0,0 +1,11 @@
+import './App.css';
+import Calculator from './components/calculator';
+
+function App() {
+  const newCalculator = new Calculator();
+  return (
+    newCalculator.render()
```

Comment:
- [ ] Please, the calculator should be called according react components format, on this case, inside return, add `<Calculator />` and you'll have your component inside app.js 🌵

Reply:
Great, you are right, now I fixed this issue. 😄